monkey/infection_monkey/monkey.py

```diff
@@ -87,6 +93,9 @@
 logging.getLogger("urllib3").setLevel(logging.INFO)
 
 
+PLUGIN_DIR = Path(gettempdir()) / "plugins"
```

Comment:
This is a security vulnerability. See https://salvatoresecurity.com/the-many-perils-of-tmp/

If you create a directory in `/tmp`, its name needs to be unguessable.

Reply:
Whoops. I meant to create a random folder name in `/tmp`. Would that be preferable, or should we choose a location outside of `/tmp`?

Reply:
Ideally, we'd use something outside of `/tmp`, but that may not always be possible since the Agent doesn't know where it's going to end up, as what user, and what that user's permissions will be. I think we're stuck with `/tmp` for now. Just choose a sufficiently long random name.